### PR TITLE
[ADD] purchase_order_manual_close: new module to finish in process purchase order

### PR DESCRIPTION
**Pull request by agaldona**

### CODE REVIEW

File: purchase_order_manual_close/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+{
+    "name": "Purchase Order manual close",
+    "version": "1.0",
+    'author': 'OdooMRP team',
+    'website': "http://www.odoomrp.com",
+    'contributors': ["Ainara Galdona <ainaragaldona@gmail.com>",
+                     "Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>",
+                     "Ana Juaristi <ajuaristio@gmail.com>"],
+    "depends": ['purchase'],
+    "category": "Custom Modules",
```
Comment (oihane) [inline purchase_order_manual_close/__openerp__.py]: `Purchase Management`

File: purchase_order_manual_close/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Ainara Galdona - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    "name": "Purchase Order manual close",
+    "version": "1.0",
+    'author': 'OdooMRP team',
```
Comment (oihane) [inline purchase_order_manual_close/__openerp__.py]:
```
"OdooMRP team, "
"AvanzOSC, "
"Serv. Tecnol. Avanzados - Pedro M. Baeza",
```

File: purchase_order_manual_close/README.rst [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+ - If the order has some invoice not finished.
+
+Credits
+=======
+
+Contributors
+------------
+* Ainara Galdona <ainaragaldona@avanzosc.es>
+* Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
+* Ana Juaristi <ajuaristio@gmail.com>
```
Comment (oihane) [inline purchase_order_manual_close/README.rst]: `anajuaristi@avanzosc.es`

File: purchase_order_manual_close/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    'website': "http://www.odoomrp.com",
+    'contributors': ["Ainara Galdona <ainaragaldona@gmail.com>",
+                     "Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>",
+                     "Ana Juaristi <ajuaristio@gmail.com>"],
```
Comment (oihane) [inline purchase_order_manual_close/__openerp__.py]: `anajuaristi@avanzosc.es`

**Comment by agaldona** [comment]
Cambios hechos :wink:

File: purchase_order_manual_close/README.rst [hunk 5]
```diff
@@ -0,0 +1,20 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+===========================
+Purchase order manual close
+===========================
+
+This module allows to force manually ending of purchase orders.
+The wizard will raise an error. 
+ - If the order is in state 'draft', 'cancel' or 'done.
```
Comment (oihane) [inline purchase_order_manual_close/README.rst:9]: There is one `'` missing.

**Pull request merged**